src/open_producten/conf/base.py

```diff
@@ -110,6 +122,8 @@
 #
 REST_FRAMEWORK = {
     "DEFAULT_AUTHENTICATION_CLASSES": [
+        "open_producten.utils.oidc_drf_middleware.OIDCAuthentication",
+        "rest_framework.authentication.SessionAuthentication",
```

Comment:
Is `SessionAuthentication` needed?

Reply:
https://mozilla-django-oidc.readthedocs.io/en/stable/drf.html they add it in the docs but it seems to work without. it seems to be needed for the browsable api (DRF) https://github.com/mozilla/mozilla-django-oidc/issues/265 but that is not enabled in OP.